### ISSUE TITLE
Please help me how to rectify this:MisconfigurationException: The provided lr scheduler `StepLR` doesn't follow PyTorch's LRScheduler API. You should override the `LightningModule.lr_scheduler_step` hook with your own logic if you are using a custom LR scheduler.

### ISSUE THREAD

**Issue by Harish11999** [Nixtla/neuralforecast#509]
![image](https://user-images.githubusercontent.com/108693125/230360409-11833824-4f3e-4309-9432-7aacc3141aab.png)

**Comment by FedericoGarza** [comment]
hey @Harish11999! Thanks for using neuralforecast. The problem is related to the latest version of torch (v2.0.0). You can solve the issue by installing the latest version of neuralforecast from the main branch. Here's the line:  

```
pip install "git+https://github.com/nixtla/neuralforecast.git@main"
```

We are preparing an official release to pip soon. :)

**Comment by Harish11999** [comment]
![image](https://user-images.githubusercontent.com/108693125/230763686-c9316613-aebf-4ded-8884-514baae7e628.png)

I'm getting error again. The models will run but while we use ".fit", there is an error. I tried installing pytorch 1.6.5 but it doesn't exist.

**Comment by kyr0** [comment]
You can pin to 1.3.0, then it will work in colab if you're not using the latest features added in 1.4.0+

> !pip install neuralforecast==1.3.0 datasetsforecast

**Comment by Harish11999** [comment]
Thank you. Now its fine
